staging/kos/deploy/main/30-role-ipam.yaml

```diff
@@ -7,21 +7,30 @@ rules:
   - network.example.com
   resources:
   - subnets
+  - networkattachments
   verbs:
   - get
```

Comment:
`get` is never used, neither for subnets nor for NAs.

Reply:
I'm not going to worry about this, the privilege to list is granted and more powerful.